### ISSUE TITLE
Prespectra as a comma category

### ISSUE THREAD

**Issue by Alizter** [mikeshulman/HoTT#36]
I'll make note of this here in case I forget about it. We can construct the wild category of prespectra as a comma category of a functor F : (forall n:N, pType) -> (forall n:N, pType) where N is a successor structure and the identity functor. N ends up inducing an endofunctor on (forall n:N, pType) which takes (fun n => P n) and gives (fun n => P n.+1). F is then defined to take (fun n => P n) and give (fun n => loops (P n.+1)).

I wonder if there is a comma category like construction for equivalences instead of maps. Then it seems that spectra would be constructed this way. I also wonder what the relation with comma categories and comma categories with equivalences is. Are there other analogues of spectrification from this?

**Comment by mikeshulman** [comment]
A comma category with an isomorphism in the square instead of a transformation is called an "iso-comma object", or (when defined up to equivalence) a pseudo-pullback or bicategorical pullback.  Both comma objects and iso-comma objects should inherit "having equivalences" from their inputs.  But I wouldn't expect a general analogue of spectrification (mapping, say, a comma object to an iso-comma one).

**Comment by mikeshulman** [comment]
Actually this isn't quite right; the category of prespectra (resp. spectra) is the _inserter_ (resp. isoinserter) of the identity functor over F, not the comma category.

**Comment by Alizter** [comment]
So if a comma category is a "pullback" then the inserter is an "equalizer" right?

**Comment by mikeshulman** [comment]
As much as it pains me to say that "false implies false" can carry meaningful content, yes.  (-:

**Comment by Alizter** [comment]
What is the inserter of here exactly? It seems to be of the loop space functor composed with the successor functor and the identity functor. Isn't an equivalent definition of spectra to use suspensions instead of loops? Then it seems to me that prespectra is the inserter of the suspension functor and sucessor functor?

**Comment by mikeshulman** [comment]
Yes, the inserter of Id and shift o loops, or equivalently the inserter of susp and shift.  (But in HoTT, I'm almost positive that loops are better to work with than susp.)
